src/lib.rs

```diff
@@ -36,13 +36,13 @@ extern crate brotli;
 extern crate flate2;
 extern crate rand;
 extern crate x86intrin;
+extern crate parquet_format;
```

Comment:
maybe we can also remove some dependencies that were moved to `parquet-format-rs`?

Reply:
Yes, I will remove them. Thanks.

Reply:
Removed them, now lib.rs should contain only parquet dependencies.